core/templates/pages/topic-editor-page/topic-editor-page.component.spec.ts

```diff
@@ -84,6 +86,13 @@ describe('Topic editor page', function() {
     spyOn(TopicEditorRoutingService, 'getActiveTabName').and.returnValue(
       'questions');
     expect(ctrl.getActiveTabName()).toBe('questions');
+    expect(ctrl.isInTopicEditorTabs()).toBe(true);
+    expect(ctrl.isInPreviewTab()).toBe(false);
+  });
+
+  it('should get entity type from context service', function() {
```

Comment:
is this the correct message for the test? I don't see any function related to the context in the test.

Reply:
Fixed.